zap/src/main/java/org/parosproxy/paros/core/scanner/AppParameter.java

```diff
@@ -0,0 +1,41 @@
+package org.parosproxy.paros.core.scanner;
+
+/**
+ * {@code AppParameter} class wraps the parameters which are used to modify the {@code HttpMessage}.
+ * It is specifically used for updating multiple parameters of {@code HttpMessage}
+ *
+ * @author preetkaran20@gmail.com KSASAN
+ */
+public class AppParameter {
```

Comment:
`AppParameter` is misleading, not just about the target app. `InputVector` would be closer to what it is.